### ISSUE TITLE
really?

### ISSUE THREAD

**Issue by thekjohn** [petikk/ZyHelperFix#2]
Are you f..g kidding me?

**Comment by petikk** [comment]
This is no insult or anything in your direction.
I am a big fan of your work and I am thankful for it, but I think we just are divided over how macros influence the game.
You can also add me on discord if you want to discuss this.

**Comment by thekjohn** [comment]
I don't believe we are divided over how macros affect the game. In fact I don't care about macro users, hackers, crackers, botters, etc. However I care about HUD and the possible banwave caused by plugins which allow people to bot/macro using HUD. If I don't act against automation plugins with EVERYTHING I can fight with, then I'll be responsible if Blizz bans HUD users telling everybody HUD supports automation.

**Comment by petikk** [comment]
I don't think blizzard decides to ban HUD users because of automation, they already stated using HUD itself could/will risk in a ban.
I see your point, but even you took more than 2 months to find this repo, even tho it was public knowledge and it has really alot of downloads. Imagine how long blizz takes.
I hope you don't take this personally, but I am tired of spamming keys like an idiot.

**Comment by malvarenga123** [comment]
> I don't believe we are divided over how macros affect the game. In fact I don't care about macro users, hackers, crackers, botters, etc. However I care about HUD and the possible banwave caused by plugins which allow people to bot/macro using HUD. If I don't act against automation plugins with EVERYTHING I can fight with, then I'll be responsible if Blizz bans HUD users telling everybody HUD supports automation.

I think you shot yourself in the foot when you disabled other programs from reading the screen. Not only you made people look for loopholes in the plugin system, which they found, and write plugins that can pass info to other programs, but you also shifted the automation focus from AHK to TH.
This crusade of yours against automation is meaningless while botting is rampant and nothing is done against it.
In this issue, just like Blizzard, you seem out of touch with your users.
